### PR TITLE
VAN-2096 fix jq setup and make usage consistent

### PR DESCRIPTION
**Pull request by kanaksinghal**
On aws pipelines, jq seems already installed but it was jq 1.5
while we were upgrading it to 1.6 (for consistency)
but the PATH variable was set in a way that the newly installed one
was getting overridden.
So we fix it by overriding the previous one.

Also, the format `jq -aRs` is not backward compatible so changing it to
`jq -aRs .` since that makes it more resilient and consistent
to the other occurrences

### CODE REVIEW

File: pipeline-modules/deployment-service/aws/basic-eks-regional.yaml [hunk 1]
```diff
@@ -166,7 +166,7 @@ template: |
           {% endfor %}
           # aws s3 rm  s3://{{customer_bucket}} --recursive ;
           {% endif %}
-          rm -f values.yaml;
+          rm values.yaml;
```
Comment (dshelley66) [inline pipeline-modules/deployment-service/aws/basic-eks-regional.yaml:169]: can you put back the -f? this will eliminate a spurious error from the log

Comment (dshelley66) [reply pipeline-modules/deployment-service/aws/basic-eks-regional.yaml:169]: Actually can you add the "-f" to any of the modules that have that "rm values.yaml"

**Comment by kanaksinghal** [comment]
see test results at: https://github.com/cldcvr/vanguard-api/pull/824/checks?check_run_id=4384270596

**Review by dshelley66 — APPROVED** [review]

**Comment by kanaksinghal** [comment]
this also passed the automation-tests.

**Pull request merged**